### PR TITLE
Fix hyperlink within Release Managers page

### PR DESCRIPTION
**Pull request by kadtendulkar**
Follow up on #34647

Updated the page [Release Managers](https://kubernetes.io/releases/release-managers/)

### CODE REVIEW

**Comment by sftim** [comment]
/retitle Fix hyperlink within Release Managers page

**Comment by Sea-n** [comment]
/lgtm cancel

The link is broken.
https://deploy-preview-34648--kubernetes-io-main-staging.netlify.app/releases/release-managers/#release-managers

**Comment by reylejano** [comment]
I concur with @Sea-n that the link is no longer a link
https://deploy-preview-34648--kubernetes-io-main-staging.netlify.app/releases/release-managers/

File: content/en/releases/release-managers.md [hunk 1]
```diff
@@ -212,7 +212,7 @@ Example: [1.15 Release Team](https://git.k8s.io/sig-release/releases/release-1.1
 [handbook-packaging]: https://git.k8s.io/sig-release/release-engineering/packaging.md
 [handbook-patch-release]: https://git.k8s.io/sig-release/release-engineering/role-handbooks/patch-release-team.md
 [k-sig-release-releases]: https://git.k8s.io/sig-release/releases
-[patches]: /patch-releases.md
+[patches]: https://kubernetes.io/releases/patch-releases/
```
Comment (reylejano) [inline content/en/releases/release-managers.md:215]:
```suggestion
[patches]: /releases/patch-releases/
```
relative path for links

**Comment by reylejano** [comment]
Thank you for fixing the link
/lgtm

**Pull request merged**